src/lamp_py/ingestion/glides.py

```diff
@@ -88,7 +89,7 @@ def append_records(self) -> None:
 
         joined_ds = new_dataset
         if os.path.exists(self.local_path):
-            joined_ds = pd.dataset([pd.dataset(self.local_path), new_dataset])
+            joined_ds = pd.dataset([new_dataset, pd.dataset(self.local_path)])
```

Comment:
swapped the order here so that "newer" data will be the one thats used when i do the unique later on.